scripts/README.md

```diff
@@ -25,6 +25,21 @@ For Slack markdown formatting, run it with:
 
     npm run release-notes <pt label> slack
 
+### Jira Version
+
+To run the Jira version of this script, you need a [Jira personal access token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html). Add the token to a `.env` file in the scripts folder with
+
+    JIRA_TOKEN=<token>
+    JIRA_USER=<your jira account email address>
+
+For GitHub markdown formatting, run it with:
+
+    npm run release-notes-jira <jira project key> <jira label>
```

Comment:
I would add an example here to show what the `<jira project key>` and `<jira label>` looks like to void confusion.